### ISSUE TITLE
Multitor directory: ownership bug

### ISSUE THREAD

**Issue by Skarlett** [trimstray/multitor#3]
after running setup.sh, and attempting to run the program i was greeted with no std output. Digging around the logs, found the issue. should be handled in setup.sh

it didn't make the directory `/var/lib/multitor`, so I made it, then it complained about a ownership problem.
to fix that I simply used `chown $USER /var/lib/multitor`

**Comment by trimstray** [comment]
Thank you for reporting the problem!

I have a few questions:

- you can restore the permissions, execute the script and send a log files to trimstray@gmail.com? It is located in the `log` directory.
- do you execute a command with the argument `-u`? If so, which user are you? Does this user exist on the system?

**Comment by Skarlett** [comment]
Thankfully the log file is tiny and shouldn't be a problem placing it here. I've reanimated my install. (previously was on a ubuntu machine, now on Arch - Same problem occured.)
Here is a full instruction of my install and commands I used, including the log file. 
(After cloning)

    $ sudo ./setup.sh install
    Create symbolic link to /usr/local/bin
    Create man page to /usr/local/man/man8
    $ multitor --init 2 -u $USER --socks-port 9000 --control port 9900
    (Blank stdout output)
    $ cat logs/stdout.log
    mkdir: cannot create directory ‘/var/lib/multitor’: Permission denied
    find: ‘/var/lib/multitor/*’: No such file or directory
    $ sudo mkdir /var/lib/multitor; sudo chown $USER -R /var/lib/multitor
    $ multitor --init 2 -u $USER --socks-port 9000 --control port 9900
     ... And it works.
[multitor runtime logs](https://gist.github.com/Skarlett/aab358d5f89e84ac9eb7b4fa878ffb6d)

**Comment by trimstray** [comment]
The basic solution to Your problem is to run the tool from the **root** user and to indicate to it the appropriate **tor** user - eg. in Debian GNU/Linux it is usually the **debian-tor**.

The structure of newly created directories should look similar to the one below:
```bash
ls -al /var/lib/multitor     
razem 28
drwx------  7 debian-tor root 4096 maj  2 08:42 .
drwxr-xr-x 98 root       root 4096 mar 20 22:00 ..
drwx------  2 debian-tor root 4096 maj  2 09:07 9000
drwx------  2 debian-tor root 4096 maj  2 09:07 9001
drwx------  2 debian-tor root 4096 maj  2 09:07 9002
drwx------  2 debian-tor root 4096 maj  2 09:07 9003
drwx------  2 debian-tor root 4096 maj  2 09:07 9004
```

In Your example, I see that You pass the name of the currently running user as parameter. This is an interesting way and in one of the next versions I will consider that there is full support for less privileged users (of course, it has some advantages and disadvantages).

At this time, I added to force the tool to run from the **root** user because:

- it has the power to fully support all udp/tcp ports
- it has the right to fully support additional services (haproxy, polipo), including management from the level of systemctl

I hope it will not be a problem. Let me close this issue but I will additionally tag it with "_Status: Feedback_" for further consultation with You. The above changes are available in the new version: **[v1.2.2](https://github.com/trimstray/multitor/releases/tag/v1.2.2)**
